docker/Dockerfile

```diff
@@ -13,6 +13,11 @@ COPY package.json pnpm-lock.yaml ./
 
 RUN pnpm install
 
+ARG NEXT_PUBLIC_GOCARDLESS_ENABLED
```

Comment:
thing is i'm hosting prebuilt docker images and this should not be a part of docker image, rather should come run time!

Reply:
I will it to a server env instead as you suggested in other comment :)

Reply:
https://github.com/oss-apps/split-pro/pull/132/commits/25fc9faeefd3aadd0b30c1b626ccec66c3506bcb